EnglishFilter/sections/content_blocker.txt

```diff
@@ -1196,8 +1194,6 @@ topbusiness24.ru,mydomsam.ru,hubuhu.ru,moysadinfo.ru,mybusinessplus.ru,dukand.ru
 @@||widgets.outbrain.com/images/$image,domain=afreesms.com
 ! https://github.com/AdguardTeam/AdguardFilters/issues/68913
 @@||dongknows.com^$generichide
-! https://github.com/AdguardTeam/AdguardFilters/issues/68815
-@@||ad.a-ads.com/*?size=$domain=getitall.top
```

Comment:
It seems that I added a rule with wrong domain in this commit - https://github.com/AdguardTeam/AdguardFilters/commit/26b7d0337e2240f16f594526b069b78d34f63949#diff-5d33fa79f5277920586720bffd6c91c26924c4cbeedfc0907448ac69e6b6e656R1192

I will fix it.